src/plugins/lightscript.js

```diff
@@ -143,7 +202,7 @@ pp.parseWhiteBlock = function (allowDirectives?, isExpression?) {
   // must start with colon or arrow
   if (isExpression) {
     this.expect(tt.colon);
-    if (!this.isLineBreak()) return this.parseMaybeAssign();
+    if (!this.isLineBreak()) return this.parseWhiteBlockBody(node, indentLevel, true, false);
```

Comment:
One possibility is that it will allow things like `break` and `continue` where they don't make any sense, but maybe that is not a parser issue?